gp-advanced-select/gpadvs-preserve-choice-order.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Gravity Perks // Advanced Select // Preserve Choice Order
+ * https://gravitywiz.com/documentation/gravity-forms-advanced-select/
+ *
+ * Preserve Choice Order with Multi Select Fields using GP Advanced Select.
+ *
+ * Instruction Video: https://www.loom.com/share/d2ad3fea02234f02871ac9b1efe20b53
+ *
+ * Instructions:
+ *
+ * 1. Install this snippet by following the instructions here:
+ *    https://gravitywiz.com/documentation/how-do-i-install-a-snippet/
+ */
+add_filter( 'gform_save_field_value', 'preserve_choice_order_with_advanced_select', 10, 4 );
+function preserve_choice_order_with_advanced_select( $value, $lead, $field, $form ) {
```

Comment:
As a general rule, we use anonymous functions for the Snippet Library. My thinking is that the less the user has read in the code, the easier it is for them to parse and therefore less intimidating.